yarn.lock

```diff
@@ -7597,7 +7617,6 @@ string-width@^3.0.0, string-width@^3.1.0:
 string-width@^4.1.0:
   version "4.2.0"
   resolved "https://registry.yarnpkg.com/string-width/-/string-width-4.2.0.tgz#952182c46cc7b2c313d1596e623992bd163b72b5"
-  integrity sha512-zUz5JD+tgqtuDjMhwIg5uFVV3dtqZ9yQJlZVfq4I01/K5Paj5UHj7VyrQOJvzawSVlKpObApbfD0Ed6yJc+1eg==
```

Comment:
Are you perhaps using an old version of yarn?

Reply:
It looks like `volta` only looks at the `package.json` in the current direction https://github.com/volta-cli/volta/issues/378#issuecomment-501541471
`volta` is pinned to `1.22.4` in the root `package.json`, but the `package.json` doesn't specify any volta config in any of the packages.

I'll copy the configs up